### ISSUE TITLE
About the scCustomize package dependent package error

### ISSUE THREAD

**Issue by PanSX-Dr** [samuel-marsh/scCustomize#215]
Hello, first of all, thank you for developing this powerful package, which really makes the visualization of single cells much easier. But I recently had problems using this package. My seurat is V4 version, and the seuratObject is also version 4.1.4, but when I use scCustomize for visualization, I actually error that the seuratObject is 5.0, so I want to ask, the version of this seuratObject must be 5.0?

**Comment by samuel-marsh** [comment]
Hi,

Thanks for the kind words.  Yes due to some of the functions needed to make the package compatible with both old (V3/4) and new (V5) assays it now requires SeuratObject 5.0.0 but only requires Seurat v4.3.0.1 or above.  So you can update SeuratObject and keep your Seurat version.  You can double check that all of your code results are the same after updating but there shouldn't be issue given that SeuratObject defines the class and functions to access objects and not the analysis functions in Seurat itself.

Best,
Sam

**Comment by PanSX-Dr** [comment]
Thank you very much for your timely reply. I have tried this method, because I have updated seuratObject before, but seurat always uses V4 version, but when seuratOjbject is updated, Matrix must be updated to 1.7 version. Then, a new problem appeared. When I ran runUMAP later, I was told that Matrix version did not match and Matrix could not be provided. So I had to return the lower version. Version 1.1 of scCustomize was also installed. Finally, thank you again for your timely reply.
